docs/sdk.rst

```diff
@@ -264,59 +264,31 @@ This also applies when starting a Globus Compute endpoint.
 .. note:: Globus Compute clients and endpoints will use the client credentials if they are set, so it is important to ensure the client submitting requests has access to an endpoint.
 
 
-.. _login manager:
+.. _existing-token:
 
-Using a Existing Tokens
+Using an Existing Token
 -----------------------
 
-To programmatically create a Client from tokens and remove the need to perform a Native App login flow you can use the *AuthorizerLoginManager*.
-The AuthorizerLoginManager is responsible for serving tokens to the Client as needed and can be instantiated using existing tokens.
+To create a |Client|_ from an existing access token and skip the interactive login flow, you can pass an |AccessTokenAuthorizer|_
+via the ``authorizer`` parameter:
 
-The AuthorizerLoginManager can be used to simply return static tokens and enable programmatic use of the Client.
-
-.. note::
-    Accessing the Globus Compute API requires the Globus Auth scope:
-
-    .. code-block:: text
-
-      https://auth.globus.org/scopes/facd7ccc-c5f4-42aa-916b-a0e270e2c2a9/all
-
-    This is also programmatically available as the ``FUNCX_SCOPE`` attribute on
-    the ``Client`` class:
-
-    .. code-block:: python
-
-      >>> from globus_compute_sdk import Client
-      >>> Client.FUNCX_SCOPE
-      'https://auth.globus.org/scopes/facd7ccc-c5f4-42aa-916b-a0e270e2c2a9/all'
-
-More details on the Globus Compute login manager prototcol are available `here. <https://github.com/globus/globus-compute/blob/main/compute_sdk/globus_compute_sdk/sdk/login_manager/protocol.py>`_
-
-
-.. code:: python
+.. code-block:: python
 
   import globus_sdk
-  from globus_sdk.scopes import AuthScopes
```

Comment:
As a simpler example, this updated code is much more easily digestable.  Well done.